### ISSUE TITLE
Delete old (unused) branches 

### ISSUE THREAD

**Issue by deni-zen** [deni-zen/csvelte#68]
There are a bunch of old branches cluttering things up in the repo. Delete all but master, dev, v0.2.0/master (this will be deleted soon but not now), and whatever new branches I create after writing this ticket. Here is a list of branches to delete:
- [x] features/47
- [x] features/46
- [x] features/42
- [x] features/17
- [x] features/11
- [x] hawkin-it

Delete these branches too but only once you pull all the code you plan to still use from them:
- [x] conform/CSVW - may contain schema, metadata, etc. code that I still may want to use 
- [x] v0.2.0/master - contains all the classes I deleted for the v0.1.0 push. Only delete this branch once you're sure you don't need any of its classes/tests

Make the release branches consistently named:
- [x] releases/v0.1
- [x] releases/v0.2

Eventually I'll also want branches for major version's concurrent development, so something like:
- v1.x/dev
- v1.x/master 

~~Also, while you're at it, I think it would actually make sense to change the "default branch" to "dev" rather than "master" since that's where the actual development happens. ~~

**Comment by nozavroni** [comment]
I deleted everything I could find that didn't have commits AHEAD of master. If there is anything I deleted that I may eventually want back I can get it from the history in the master branch. I'll leave the branch naming for new versions stuff for later (crossed bridges and such)
